hera_opm/mf_tools.py

```diff
@@ -392,6 +559,37 @@ def prep_args(args, obsid, pol=None, obsids=None):
         else:
             args = re.sub(r"\{next_basename\}", oids[obs_idx + 1], args)
 
+    if re.search(r"\{obsid_list\}", args):
+        if time_centered is None:
+            time_centered = True
+        try:
+            n_time_neighbors = int(n_time_neighbors)
+        except ValueError:
+            raise ValueError(
+                "n_time_neighbors must be able to be interpreted as an int."
+            )
+        try:
+            stride_length = int(stride_length)
+        except ValueError:
+            raise ValueError("stride_length must be able to be interpreted as an int.")
+        obsids = sort_obsids(obsids)
+        obs_idx = obsids.index(obsid)
+        # Compute the number of remaining obsids to process.
```

Comment:
Evidently this works, but is it possible/less error-prone to reuse the output of `_determine_stride_partitioning`?